### PR TITLE
[4.0] Fix typos

### PR DESCRIPTION
**Pull request by HLeithner**
Just 2 typos

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on d8e7c1281315b3ccd64576239a8b987c6ce5e8ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28703">issues.joomla.org/tracker/joomla-cms/28703</a>.</sub>

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on d8e7c1281315b3ccd64576239a8b987c6ce5e8ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28703">issues.joomla.org/tracker/joomla-cms/28703</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28703">issues.joomla.org/tracker/joomla-cms/28703</a>.</sub>

File: libraries/src/Toolbar/Button/DropdownButton.php [hunk 1]
```diff
@@ -104,7 +104,7 @@ protected function renderButton(array &$options): string
 	 *
 	 * @return  string  Button CSS Id
 	 *
-	 * @since   3.0
+	 * @since   4.0.0
```
Comment (wilsonge) [inline libraries/src/Toolbar/Button/DropdownButton.php:107]: There's several other classes this deploy version for this function needs to be fixed in: https://github.com/joomla/joomla-cms/commit/9e3bd1a91fc2813f7a4d20f37f9a54dc4032bc21

Comment (richard67) [reply libraries/src/Toolbar/Button/DropdownButton.php:107]: @wilsonge Shall I remove RTC and set label "Updates Requested"?

**Comment by wilsonge** [comment]
Thanks!

File: libraries/src/Toolbar/Button/DropdownButton.php (hunk 1 above)
Comment (wilsonge) [reply libraries/src/Toolbar/Button/DropdownButton.php:107]: Fixed the last one manually just now

**Pull request merged**